### PR TITLE
Add mock link to the table

### PR DESCRIPTION
**Pull request by alejandrocabriales**
## Ticket
https://trello.com/c/1BmBP42a/330-bsn-1-budget-summary-navigation-list-of-issues

## Description
Fix issues in BS + Navigation

## What solved
- [X] : FR The sub-categories should be clickable and navigate to their page.

### CODE REVIEW

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx [hunk 1]
```diff
@@ -44,44 +47,50 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
       {showFooterAndCorrectNumber.map((table: TableFinances, index) => (
         <TableContainer isLight={isLight} className={className} key={index} hasOthers={table.others || false}>
           <TableBody isLight={isLight}>
-            {table.rows.map((row: ItemRow, index) => (
-              <TableRow key={index} isLight={isLight} isMain={row.isMain}>
-                <Headed isLight={isLight} period={period}>
-                  {row.name}
-                </Headed>
+            {table.rows.map((row: ItemRow, index) => {
+              console.log('row.codePath', row.codePath?.replace('atlas', ''));
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx:51]: did you forget to remove this `console.log`?

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx [hunk 2]
```diff
@@ -44,44 +47,50 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
...
+              const href = `${siteRoutes.finances((row.codePath ?? '').replace('atlas/', ''))}?year=${year}`;
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx:52]: please, note that in the first level the code path is `atlas` so it is not going to be replaced as `atlas !== atlas/` so you should handle this in order to get the right link

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx:51]: done

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx:52]: Sorry I don't got your point, the links start from that level and then going down, so I don't think that affect the navigations, but not sure if I got your point

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**